src/github.com/stellar/horizon/main_generated.go

```diff
@@ -90,6 +90,13 @@ func (action OrderBookShowAction) ServeHTTPC(c web.C, w http.ResponseWriter, r *
 }
 
 // ServeHTTPC is a method for web.Handler
+func (action OrderBookTradeIndexAction) ServeHTTPC(c web.C, w http.ResponseWriter, r *http.Request) {
+	ap := &action.Action
+	ap.Prepare(c, w, r)
+	ap.Execute(&action)
```

Comment:
Yeah, I can certainly improve the action system (I didn't have enough experience with go when I came up with the existing solution), but I'm hesitant to invest in a solution at this moment since I'm planning on moving our standard mux over to https://github.com/go-chi/chi.

I'm planning on revisiting this problem for our monorepo-based common solution, and I'll make sure to put whatever I come up with in front of you.